R/foreign-keys.R

```diff
@@ -152,31 +152,34 @@ cdm_rm_fk <- function(dm, table, column, ref_table) {
 
 #' Find foreign key candidates in a table
 #'
+#' Which columns are candidates for a foreign key of a table, referencing
+#' the primary key column of another [`dm`] object's table?
+#'
 #' @inheritParams cdm_add_fk
 #' @param table The table whose columns should be tested for foreign key candidate potential
 #' @param ref_table A table with a primary key.
 #'
-#' @description Which columns are foreign candidates of a table, referencing the primary key column of another [`dm`] object's table?
-#' `cdm_enum_fk_candidates()` checks first, if `ref_table` has a primary key set. Then it determines
-#' for each column of `table`, if this column contains only a subset of values of the primary key column of
-#' `ref_table` and is therefore a candidate for a foreign key from `table` to `ref_table`.
+#' @details `cdm_enum_fk_candidates()` checks first, if `ref_table` has a primary key set. Then
+#' it determines for each column of `table`, if it has the same class as the primary key of `ref_table`.
+#' If this is the case a check is performed, if the column contains only a subset of values of the
+#' primary key column of `ref_table`. If this is `TRUE`, this column is a candidate for a foreign key
```

Comment:
Can you use bullets or an enumeration to describe the rules?

```suggestion
#' primary key column of `ref_table`.
#' This check uses [anti_join()] which implicitly checks type compatibility.
#' If this is `TRUE`, this column is a candidate for a foreign key
```